### DISCUSSION TITLE
Technicolor dja0231tls sip problem?

### DISCUSSION THREAD

**Discussion by lindsay1974** [hack-technicolor/hack-technicolor#311] [General]
--redacted--

**Comment by lindsay1974** [comment]
I have successfully used the hack for the dja0231 vant v on my second router (modem) plugged into my network
My network runs on a non hacked dja0231 vant v router (modem) supplied by telstra,
Basically primary dja0231 = connect to internet phone sip ect
secondary dja0231 hacked, used as a sub branch off that
 How do I get the secondary sub branch to forward the sip network on to the primary?
would appreciate any ideas, this is doing my head in

cheers 
Linz

**Comment by lindsay1974** [comment]
anybody with assistance would be gratefully accepted

**Comment by LuKePicci** [comment]
Please repost this as discussion in the appropriate section. The model support request template is about getting new technicolor models supported by these guides. Thanks for understanding
